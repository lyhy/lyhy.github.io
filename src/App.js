import React from 'react';
import {Helmet} from "react-helmet";
import './App.css';
import Nav from './days/components/Nav/Nav'

function App() {
  return (
    <div>
      <Helmet>
          <title>Learning React</title>
          <meta
            name="viewport"
            content="minimum-scale=1, initial-scale=1, width=device-width"
          />
          <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
          <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />

      </Helmet>
      <header>
        <Nav></Nav>
      </header>
    </div>
  );
}

export default App;
