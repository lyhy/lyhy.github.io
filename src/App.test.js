import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

test('renders learn react link', () => {
  const { getAllByText } = render(<BrowserRouter>
    <App/>
  </BrowserRouter>);
  const linkElement = getAllByText(/Home/i);
  expect(linkElement[0]).toBeInTheDocument();
});

test('snapshot', () => {
  const component = renderer.create(<BrowserRouter>
    <App/>
  </BrowserRouter>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
