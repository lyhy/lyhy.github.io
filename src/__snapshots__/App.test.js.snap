// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <div>
    <ul>
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Home
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          href="/page1"
          onClick={[Function]}
        >
          Repo
        </a>
      </li>
    </ul>
    <div>
      Home
    </div>
  </div>
</div>
`;
